source/docs/v3/console-commands.blade.md

```diff
@@ -9,7 +9,7 @@ section: content
 
 The package comes with some useful artisan commands.
 
-Tenant-aware commands run for all tenants by default. The commands also have the `--tenants` option which lets you specify IDs of the tenants for which the command will run.
+Tenant-aware commands run for all tenants by default. The commands also have the `--tenants` option which lets you specify keys of the tenants for which the command will run.
```

Comment:
I would still keep IDs here even if the package uses *keys* internally because IDs are intuitive and people new to the package will understand that easier than "keys"

Reply:
Same for the other change. We typically only use "id" in the docs I believe. Even if the column can technically be changed.

Reply:
> IDs are intuitive and people new to the package will understand that easier than "keys"

I see, that's probably right